packages/store/src/internal/dispatcher.ts

```diff
@@ -32,7 +33,7 @@ export class InternalDispatcher {
   /**
    * Dispatches event(s).
    */
-  dispatch(actionOrActions: any | any[]): Observable<any> {
```

Comment:
What is the reason for making this change as part of this PR?

Reply:
I actually was going through `dispatch` chain of calls and found that argument name is inconsistent everywhere. In some places it's called `event`, in some places it's called `action`, but it's actually `actionOrActions`, since those methods accept `actionInstance || Array<actionInstance>`.
We will never have another reason to change that, because no one will make a PR just to change the argument name :)
My hands just have reached there and from my perspective I thought it's better to keep naming consistent.

Reply:
Ok. I'm happy with the parameter name change, but please keep the type to be `any | any[]`.
Even though it is really no different with type checking, it still documents the API well.

Reply:
Hmmm, might be. The reason why I changed that to `any` because any type unit with `any` always default to `any`. I can revert it back to `any | any[]` if you wish.